.changeset/sweet-ladybugs-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/backend': patch
+---
+
+Add `password_hasher` and `password_digest` to `users.updateUser(userId, params)`
```

Comment:
We will update the changeset in a separate PR to also include the `UserMetadataParams`.